popdat/src/import_census.rs

```diff
@@ -1,11 +1,126 @@
-use map_model::Map;
-
 use crate::CensusArea;
+use abstutil::Timer;
```

Comment:
nit: For consistency, we should still split the imports based on category... std, external crates, other crates in this workspace, the current crate. I guess we don't have any contributors using an IDE that forces this order, so not a huge deal either way.

Reply:
happy to abide